docs/UserGuide.adoc

```diff
@@ -72,6 +72,11 @@ Examples:
 Shows a list of all persons, along with their non-private details, in the address book. +
 Format: `list`
 
+== Sorted List : `sort`
+
+Shows a list of all persons sorted in ascending order, along with their non-private details, in the address book. +
```

Comment:
That's good that you have updated the User Guide (UG) to show the newly implemented function that you have created. The UG is the place where the user will go to find out the different features and the usage of it most of the time.